docker-compose.yml

```
@@ -11,8 +11,8 @@ x-args:
 services:
   buildx:
     build:
-      context: ./bay/images
-      dockerfile: $PWD/bay/images/Dockerfile.buildx
+      context: ./bay/images/bay-buildx
```

Need to remove the `bay` part of this filepath. Same issue appears in lots of spots in this field.

```suggestion
      context: ./images/bay-buildx
```

---

Thanks Nick, I have removed Bay dir from the path.